widget_driver/coverage/lcov.info

```diff
@@ -0,0 +1,96 @@
+SF:lib/src/utils/platform_environment.dart
```

Comment:
Do we really want this file checked in to version control? 
I would suggest not to do so and instead add the whole **widget_driver/coverage/** folder to our .gitignore file.

Reply:
Good idea